src/storage/eventsCache/InMemory.js

```diff
@@ -10,17 +11,9 @@ class EventsCache {
         this.store = store;
         const settings = store.get(CONFIGURATIONS);
         const events = store.get(EVENTS);
-
-        this.onFullQueue = false;
         this.maxQueue = settings.size.eventsQueueSize;
-        if (events && events.length > 0) {
```

Comment:
This also should not be removed because of the fact that I am not having different storage for different client objects, it's just that I am using same storage for different client so checking if there are any events left from prev client obj that needs to be flushed.

Reply:
Thanks Mahrukh, I was thinking EventsCache is created per sdk.